cloud_formation/configs/backup.py

```diff
@@ -17,18 +17,18 @@
 and DynaamoDB tables to S3 (protected by SSE-S3 data at rest).
 """
 
+DEPENDENCIES = ['core', 'api']
```

Comment:
Is this used?

Reply:
You mean the backup config? Not right now. It has been so long since it was created that we need to update it to use new AWS features. We do need this capability, as it allows us to delete all of the CloudFormation templates and rebuild a stack without data loss.